build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -41,7 +41,15 @@ set -xeu -o errtrace
 
 this_dir="$(cd $(dirname $0) && pwd)"
 script_name="$(basename $0)"
-repo_root="$(cd "${this_dir}" && git rev-parse --show-toplevel)"
+
+# We avoid using git rev-parse --show-toplevel here since some filesystem
```

Comment:
We use git in all our other scripts. I'd prefer not to make this one any more special than it already is. I'm probably not going to be able to get to this till the new year, but I'd prefer to wait rather than hotfixing. You should be able to make whatever changes you need in the shark fork.

Reply:
No worries. We will wait as much as possible before we fork this. Our patchset is growing (~40 patches) and I want to shrink that gap. We can revisit in the new year and fork if it gets delayed in the new year.